### PR TITLE
fix(k8s): improve kubernetes deployment

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Improve kubernetes deployment by adding permission to get namespaces and changing the image to the official one.
Also group the files in `kubernetes` folder.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: kubernetes/job.yaml [hunk 1]
```diff
@@ -10,10 +10,10 @@ spec:
     spec:
       containers:
       - name: prowler
-        image: docker.io/prowler/kubernetes
+        image: toniblyx/prowler:latest
```
Comment (jfagoagas) [inline kubernetes/job.yaml]: Wouldn't be better to use `stable`? Latest is a development version.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: kubernetes/job.yaml (hunk 1 above)
Comment (MrCloudSec) [inline kubernetes/job.yaml]:
```suggestion
        image: toniblyx/prowler:stable
```

Comment (MrCloudSec) [reply kubernetes/job.yaml]: Sure, thanks!

**Review by jfagoagas — APPROVED** [review]
Great one, thanks @sergargar 👏

**Pull request merged**